proposals/views/talks.py

```diff
@@ -31,13 +32,59 @@ def get_queryset(self):
         return super().get_queryset().filter(
             state=Proposal.PROPOSAL_ACCEPTED).order_by('?')
 
+
+    def get_tags(self, talks):
+        tags = talks.filter(tags__isnull=False).values_list('tags', flat=True).distinct().order_by()
+        tag_list = []
+        [tag_list.extend(tag) for tag in tags]
+
+        tag_options = []
+        [tag_options.append((tag, tag)) for tag in set(tag_list)]
```

Comment:
Čia tas pats matyt

Reply:
čia tai turbūt ok, čia iš listo daromas list of tuples, kad paskui kaip options formoje galima būtų naudoti.